kolibri_exercise_perseus_plugin/locale/fr_FR/LC_FRONTEND_MESSAGES/exercise_perseus_render_module-messages.json

```diff
@@ -1,8 +1,125 @@
 {
-  "perseus.showScratch":"Afficher le bloc-notes",
-  "perseus.notAvailable":"Bloc-notes indisponible",
-  "perseus.loading":"Chargement ...",
-  "perseus.hint":"Obtenir un Conseil",
-  "perseus.hintLable":"Conseil :",
-  "perseus.noMoreHint":"Ne pas afficher les conseils"
-}
+  "exercisePerseusRenderer.showScratch":"Afficher le bloc-notes",
```

Comment:
Yeah this one uses double quotes, might be good to be consistent in that regard.

Reply:
This is JSON, the other is JS. JSON only allows double quotes, but our JS linting standards prefer single quotes wherever possible.